disnake/state.py

```diff
@@ -1114,8 +1115,8 @@ def parse_thread_create(self, data: gateway.ThreadCreateEvent) -> None:
         guild._add_thread(thread)
         if not has_thread:
             if data.get("newly_created"):
-                if isinstance(thread.parent, ForumChannel):
-                    thread.parent.last_thread_id = thread.id
+                if type(thread.parent) in (ForumChannel, MediaChannel):
```

Comment:
why this over an `isinstance`?

Reply:
consistency as of how its used elsewhere